src/openrct2-ui/audio/AudioMixer.cpp

```diff
@@ -484,7 +481,7 @@ namespace OpenRCT2::Audio
             for (int32_t i = 0; i < length; i++)
             {
                 float t = static_cast<float>(i) / length;
-                data[i] = static_cast<uint8_t>(data[i] * ((1 - t) * startvolume_f + t * endvolume_f));
+                data[i] = static_cast<float>(data[i]) * ((1 - t) * startvolume_f + t * endvolume_f);
```

Comment:
```suggestion
                data[i] = static_cast<uint8_t>(data[i] * ((1.0f - t) * startvolume_f + t * endvolume_f));
```
Would this suffice @ZehMatt ?